.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,12 @@
+## What changes were proposed in this pull request?
+
+(Please fill in changes proposed in this patch)
```

Comment:
In this section, we need:
1) What is the change?
2) Why the change?
3) Summary of the change (interface and short design) - needed unless we have a gthub issue for the design below